forage-android/src/test/java/com/joinforage/forage/android/vault/RosettaPinSubmitterTest.kt

```diff
@@ -160,16 +160,19 @@ class RosettaPinSubmitterTest() : MockServerSuite() {
 
         val firstError = (result as ForageApiResponse.Failure).errors[0]
 
-        // NOTE: The Rosetta error-parsing is not perfect at the moment.
-        // We currently attempt to parse Rosetta errors as Forage errors on failed requests.
-        // Which results in an undesirable (but currently tolerable) JSON Exception
-        // This will be fixed when we introduce sync capture + balance.
-        assertEquals("JSONObject[\"path\"] not found.", mockLogger.errorLogs.last().getError()!!.message)
```

Comment:
`path` is indicative of the legacy "list errors" format that Payments API returns. Now that we have the `ErrorListResponse` class, we won't throw a JSON parse error anymore so we don't want to test that this happens